binapigen/strings.go

```diff
@@ -87,7 +89,9 @@ func usesInitialism(s string) string {
 
 // camelCaseName returns correct name identifier (camelCase).
 func camelCaseName(name string) (should string) {
-	name = strings.Title(name)
+
+	name = cases.Title(language.Zulu).String(name)
```

Comment:
Can't remember why this was needed.. vaguely remember `strings.Title` hitting some special case..

Reply:
This function was apparently deprecated to facilitate localization [0]
[0] https://pkg.go.dev/strings#Title